src/components/pages/ShopDetail.tsx

```diff
@@ -132,7 +132,7 @@ function ShopDetail() {
       })
       .catch(error => console.log("データの取得に失敗", error))
     console.log("お気に入り情報", favoriteData.id)
-  },[loggedInStatus]);
+  },[user]);
```

Comment:
デプロイすると、ここの部分で
「React Hook useEffect has missing dependencies: 'favoriteData.id', 'id', and 'user.id'. Either include them or remove the dependency array」
というエラーが出てしまいます。
この場合、何を第二引数に渡せばいいのでしょうか。

Reply:
説明通り、favoriteData.id, id, user.id を渡せば良いと思います！
```suggestion
  },[favoriteData.id, id, user.id])
```

Reply:
かしこまりました！
ありがとうございます！